packages/gatsby-plugin-mdx/src/__tests__/compile-mdx.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @jest-environment node
+ */
+
+import { readFileSync } from "fs"
+import path from "path"
+
+import { NodePluginArgs } from "gatsby"
+
+import { compileMDX, compileMDXWithCustomOptions } from "../compile-mdx"
+
+const exampleMdxPath = path.resolve(
+  __dirname,
+  `..`,
+  `__fixtures__`,
+  `example.mdx`
+)
+const exampleMdxContent = readFileSync(exampleMdxPath)
+
+const cache = {
+  directory: __dirname,
+} as unknown as NodePluginArgs["cache"]
+
+const reporter = {
+  info: jest.fn(),
+  verbose: jest.fn(),
+  warn: jest.fn(),
+  panic: jest.fn(e => {
+    throw e
+  }),
+  panicOnBuild: jest.fn(e => {
+    throw e
+  }),
+} as unknown as NodePluginArgs["reporter"]
+
+describe(`compiles MDX`, () => {
+  it(`default`, async () => {
+    console.log(exampleMdxContent.toString())
+    try {
+      await compileMDX(
```

Comment:
I get `ModuleNotFoundError: Cannot find module 'estree-walker' from 'node_modules/estree-util-build-jsx/lib/index.js'`

Super weird, `estree-walker'` is there on my FS.